src/main/java/platform/codingnomads/co/springdata/example/querydsl/QueryDSLDemo.java

```diff
@@ -74,6 +71,35 @@ public void run(String... args) throws Exception {
                 .fetchOne();
         System.out.println(area);
 
+        // Get area by id. This returns null if the id does not exist in the DB.
+        // SQL never reuses an id. re: Strong consistency
+        System.out.println("\n** Get area by id **");
+        query = new JPAQuery<>(entityManager);
+        area = query.select(qArea)
+                .from(qArea)
+                .where(qArea.id.eq(1L))
```

Comment:
We did discuss it, but I'll duplicate it here for the history. 
Be careful when it comes to `SQL` and `id`. 
`id`s are generated, so unless it is guaranteed that they exist the result of the request could be `null`.